src/nft/components/collection/CollectionStats.css.ts

```diff
@@ -134,9 +134,6 @@ export const statsLabelLoading = style([
 
```

Comment:
I believe you can remove the iconsLoading style since the social icons loading state has been removed